draft-ietf-quic-qlog-main-schema.md

```diff
@@ -1722,9 +1724,13 @@ without sufficient care might not enhance privacy or security and
 could diminish the utility of qlog data.
 
 Operators and implementers should balance the value of logged data with the
-potential risks of (involuntary) disclosure, which can depend on use cases
-(e.g., research datasets might have different requirements to live operational
-troubleshooting).
+potential risks of voluntary or involuntary disclosure to trusted or untrusted
+entities. The data requirements and definition of entities depends greatly on
```

Comment:
"data requirements" is a bit vague... which requirements specifically does this refer to?

Reply:
Discussed in editors' meeting.

Goal is to talk about data requirements of the loggers (so operators and implementers), NOT data requirements of the entities it might be disclosed. 

Robin: split up two concepts (operators+impl plus entities) somehow maybe? or at least make clearer.

Dry solution: "The data requirements of the operators/implementers, and definition of entities..."

Reply:
This is the best I've gotten in 30 minutes... :) 

> Operators and implementers should balance the value of logged data with the
potential risks of voluntary or involuntary disclosure to trusted or untrusted
entities. Importantly, both the comprehensiveness required to make the data useful and the definition of entities depend greatly on the intended use cases.

Reply:
Editors' meeting: Lucas doesn't like "comprehensiveness": replace with some synonyms :)